.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -4,7 +4,16 @@
 
 Please understand this diagram before proceeding. It explains how to get your PR approved & merged.
 
-![diagram](https://github.com/Azure/azure-rest-api-specs/assets/4429827/5e91ff2d-1f5c-4e96-87c2-390e7451da37)
+![diagram](https://github.com/Azure/azure-rest-api-specs/assets/4429827/8cb0b0e5-55ba-44a4-9848-2faead57fcc0)
+
+[1]  
```

Comment:
>[1] [](http://example.com/codeflow?start=0&length=3)

We need to indicate here - that the author needs to remove the ARMChangesRequested label after breaking changes approval to get their PR on the review queue